requirements.txt

```diff
@@ -16,7 +17,8 @@ scikit-image==0.18.3
 scikit-learn>=0.23.2
 shapely
 sphinx==4.1.2
-shapely
+sphinx_gallery
```

Comment:
do we need this outside dev environment?

Reply:
Resolved in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/157/commits/06b2d319969a284b19c4c631c72a11fd5c2dc632